scripts/functions.sh

```diff
@@ -1,24 +1,41 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
 
 gitcmd="git -c commit.gpgsign=false"
 
 color() {
-    if [ $2 ]; then
-            echo -e "\e[$1;$2m"
+    if [ -n "${2:-}" ]; then
+        printf "\e[%s;%sm" "$1" "$2"
     else
-            echo -e "\e[$1m"
+        printf "\e[%sm" "$1"
     fi
 }
+
 colorend() {
-    echo -e "\e[m"
+    printf "\e[m"
+}
+
+# Check if a string contains a substring.
+#
+# https://stackoverflow.com/a/8811800/5830060
+str_contains() {
+    string="$1"
+    substring="$(printf '%q' "$2")"
+    if [ "${string#*$substring}" != "$string" ]; then
+        return 0 # $substring is in $string
+    else
+        return 1 # $substring is NOT in $string
+    fi
 }
 
 paperstash() {
-    STASHED=$($gitcmd stash  2>/dev/null|| return 0) # errors are ok
+    # We want the exit code here to avoid "magic strings".
+    $gitcmd stash push 2>/dev/null && STASHED=0 || STASHED="$?"
 }
 
 paperunstash() {
-    if [[ "$STASHED" != "No local changes to save" ]] ; then
-        $gitcmd stash pop 2>/dev/null|| return 0 # errors are ok
+    if [ "${STASHED:-0}" ]; then
```

Comment:
This will always be true - If `STASHED` is set, it's not an empty string, and it'll pass. If it's null, it'll be replaces with string `"0"` which isn't empty, and so it'll pass.